schema/schema.go

```diff
@@ -183,10 +183,23 @@ func (ta *Table) FindColumn(name string) int {
 	return -1
 }
 
+// Get TableColumn by primary index index
 func (ta *Table) GetPKColumn(index int) *TableColumn {
+	if index >= len(ta.PKColumns) {
+		return nil
+	}
 	return &ta.Columns[ta.PKColumns[index]]
 }
 
+func (ta *Table) IsPrimaryKey(colIndex int) bool {
+	for _, _colIndex := range ta.PKColumns {
```

Comment:
maybe you can just use `i` instead of `_colIndex`, we don't have `_xxx` stype in other codes.

Reply:
done